src/components/tests/ThemeToggle.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ThemeToggle from '../ThemeToggle';
+
+describe('selector types', () => {
+  it('select the search and test if only one input is rendered', () => {
```

Comment:
hehe, it might the be result of copy-paste but I do not see the description reflects what the test does 😅